app/controllers/trainees/diversity/disclosures_controller.rb

```diff
@@ -7,9 +7,10 @@ def edit
 
       def update
         updater = Diversities::Disclosures::Update.call(trainee: trainee, attributes: disclosure_param)
+        trainee = updater.disclosure.trainee
```

Comment:
can we move that closer to usage, there is a private `trainee` method...

Reply:
done https://github.com/DFE-Digital/register-trainee-teachers/pull/119/commits/8e7814211ba680f2b9d0e1f6b510a84c046ba925, cleaned up a few other bits too